pyproject.toml

```diff
@@ -38,7 +38,11 @@ test = [
     "pytest-benchmark",
     "pytest-cov",
     "pytest-rerunfailures",
+    "pytest-timeout",
     "pytest-xdist",
+    "qiskit==1.1.2",
+    "qiskit-braket-provider==0.4.1",
```

Comment:
Is it worth making these `>=` instead of `==`, so that we might more quickly catch regressions in new versions? Otherwise we'll have to remember to come update this.

Reply:
Actually we should probably loosen them because this won't work with the NBIs (which are on older Qiskit) as is